spec/jobs/sms_broker/send_sms_job_spec.rb

```diff
@@ -6,28 +6,33 @@ module SmsBroker
     describe "#perform" do
       it "raises exception if sending of SMS fails" do
         job = SendSmsJob.new(nil)
-        SmsSender.any_instance.stub(:send) { raise }
```

Comment:
rspec turns to be a bit critical to raising "just an error", so we need to be more specific here